### DISCUSSION TITLE
Sharing with lando

### DISCUSSION THREAD

**Discussion by RhRU** [exposedev/expose#199] [Q&A]
Can you help with lando configuration.
When you type expose everything is stuck it won't load
![image](https://user-images.githubusercontent.com/4202968/105595299-af722080-5d93-11eb-85b2-aee58b898cc4.png)

No problems on valet

**Comment by StevenGFX** [comment]
Running Lando in Ubuntu on WSL2 on Windows 10. I was able to share after adding my Lando domain to /etc/hosts/ in Ubuntu. 

`127.0.0.1       myapp.lndo.site`

Hopefully this helps someone!
